docker/dev/docker-entrypoint.sh

```diff
@@ -16,19 +16,21 @@
 # limitations under the License.
 #
 
-#set -e
+# set -e
 
 # Set the SYSDIG_SKIP_LOAD variable to skip loading the sysdig kernel module
 
 if [[ -z "${SYSDIG_SKIP_LOAD}" ]]; then
```

Comment:
This symbol still has SYSDIG in it

Reply:
Yes in the plan to change it. Going to do it now